src/main/webapp/app/exercises/programming/manage/grading/tasks/programming-exercise-task/programming-exercise-task.component.ts

```diff
@@ -33,6 +33,11 @@ export class ProgrammingExerciseTaskComponent implements OnInit {
 
     ngOnInit(): void {
         this.openSubject.subscribe((open) => (this.open = open));
+
+        // If this is the only task have it open by default
```

Comment:
This comment is not needed. I suggest to remove it or, if you want to be verbose, to add variable as condition: 'const isCurrentTaskTheOnlyOne = ...;'

Reply:
Not needed strictly but not bad either. Also, there are worse comments in Artemis ;)